README.md

```diff
@@ -10,9 +10,9 @@ to the app to be solved.  The app then solves the puzzle, using OpenCV to do OCR
 solution superimposed on the original image.
 OpenCV does not run on traditional App Engine instances due to their sandboxing restrictions.
 
-Note: Currently, Managed VMs are part of the [Cloud Platform Early Access Program](https://developers.google.com/cloud/early-access), and so you will need to be part of this program, and have your app whitelisted for Managed VMs, to run this demo.
+Note: Currently, Managed VMs are in Beta and as such may be subject to backward-incompatible changes.
```

Comment:
Actually... let's just remove this line from this README.  It's indicated in the main docs.